src/Evolution/Systems/Cce/System.hpp

```diff
@@ -0,0 +1,116 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+/*!
+ * \brief The set of utilities for performing Cauchy characteristic evolution
+ * and Cauchy characteristic matching.
+ *
+ * \details Cauchy characteristic evolution (CCE) is a secondary nonlinear GR
+ * evolution system that covers the domain extending from a spherical boundary
+ * away from the strong-field regime, and extending all the way to future null
+ * infinity \f$\mathcal I^+\f$. The evolution system is governed by five
+ * hypersurface equations that are integrated radially along future null slices,
+ * and one evolution equation that governs the evolution of one hypersurface to
+ * the next.
+ *
+ * The mathematics of CCE are intricate, and SpECTRE's version implements a
+ * number of tricks and improvements that are not yet present in other contexts.
+ * For introductions to CCE generally, see papers \cite Bishop1997ik,
+ * \cite Bishop1998uk, and \cite Barkett2019uae. Here we do not present a full
+ * description of all of the mathematics, but instead just provide a high-level
+ * roadmap of the SpECTRE utilities and how they come together in the CCE
+ * system. This is intended as a map for maintainers of the codebase. If you are
+ * interested primarily in running SpECTRE CCE to obtain waveform data, please
+ * see the executable documentation.
+ *
+ * First, worldtube data from a completed or running GH evolution must be
```

Comment:
define GH

Reply:
also this should be true for any Cauchy evolution of Einstein's equations, say we have only implemented GH